app/experimenter/nimbus-ui/src/components/HeaderEditExperiment/index.test.tsx

```diff
@@ -28,4 +28,17 @@ describe("HeaderEditExperiment", () => {
       "DRAFT",
     );
   });
+  it("renders with no name when no name", () => {
```

Comment:
Is it possible for users to save an experiment without a name or is this leftover from removing the update-name-on-user-input functionality which can now be removed?